src/vhost_user/message.rs

```diff
@@ -795,8 +795,15 @@ struct DescStateSplit {
     counter: u64,
 }
 
+impl Default for DescStateSplit {
+    fn default() -> Self {
+        Self::new()
+    }
+}
```

Comment:
it would be better to move current implementation into default(), and let new() call default().